Code/GraphMol/Canon.cpp

```diff
@@ -1558,16 +1539,16 @@ class RankedValue {
   }
 };
 
-bool doesAtomChiralityVary(const OrderedSet<RankedValue> &allRankedValues,
+bool doesAtomChiralityVary(const std::set<RankedValue> &allRankedValues,
                            unsigned int index) {
-  PRECONDITION(allRankedValues.size() != 0, "bad allRankedValues size");
+  PRECONDITION(!allRankedValues.empty(), "bad allRankedValues size");
   PRECONDITION(allRankedValues.begin()->getNumChiralAtoms() > index,
                "index out of range");
 
-  auto firstChiralVal =
-      allRankedValues[0].getChiralAtoms()[index].getChiralType();
+  const auto firstChiralVal =
```

Comment:
In a previous version, allRankedValues was a vector and this used allRankedValues.front().  then it was changed to a set, which does not have front().  Use of begin() makes better sense.